cli/pipelines/publish.ttl

```diff
@@ -129,24 +142,24 @@
     ] .
 
 <#shapeURLs> a :Step   ;
-    code:implementedBy [ 
+    code:implementedBy [
       a code:EcmaScriptModule  ;
-      code:link <node:stream#Readable.from> ;     
+      code:link <node:stream#Readable.from> ;
     ] ;
      code:arguments (
       """[
-        'https://cube.link/v0.1.0/shape/standalone-constraint-constraint',
-        // 'https://cube.link/v0.1.0/shape/datacatalog-constraint',        
-      ]"""^^code:EcmaScript  
+        'https://cube.link/latest/shape/standalone-constraint-constraint',
```

Comment:
v0.1.0 is obsolete, but we really want the latest instead of a fixed, recent version?

Reply:
Good point. This should be configurable too, at some point